asset-services/asset-services-api/src/main.rs

```diff
@@ -0,0 +1,13 @@
+use std::error::Error;
+
+use asset_services_celery::run_helpers::{build_app, init_logging};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
How will we call the asset services from the outside. Is the plan to expose http endpoints for that? If so we probably just need to chat about if we are going to pull in actix-web for this as well or if we want to look at another option.

Reply:
Yup, the plan is to expose a web API here, and to use whatever's most pragmatic for that.

Reply:
Update: I added a placeholder "ping" task and endpoint using Axum.